agent/api/eni/eni.go

```diff
@@ -38,14 +40,28 @@ type ENI struct {
 	// DomainNameSearchList specifies the search list for the domain
 	// name lookup, for the eni
 	DomainNameSearchList []string `json:",omitempty"`
-
+	// InterfaceVlanProperties contains information for an interface
+	// that is supposed to be used as a VLAN device
+	InterfaceVlanProperties *InterfaceVlanProperties
```

Comment:
can you modify the String() method in this file to include the information of this field for branch eni? also this field should probably be omitted if empty